app/store/chatList.go

```diff
@@ -300,6 +302,51 @@ func (s *Sessions) CreateSessionForE164(tel string, UUID string) *Session {
 	return ses
 }
 
+func (s *Sessions) CreateSessionForUUID(UUID string) *Session {
+	contact := GetContactForUUID(UUID)
+	ses := &Session{
+		Tel:          contact.Tel,
+		Name:         contact.Name,
+		Active:       true,
+		IsGroup:      false,
+		Notification: true,
+		UUID:         UUID,
+	}
+	if s.Len == 0 {
```

Comment:
Please use meaningful variable name, it's hard for me to follow otherwise. Looks like `s` and `ses` are "Sessions", are they new, old, created, received... or something else, I don't know.

Reply:
the s I cannot change and it relates to the "s" in the function call. This function is like a class function for the type `Sessions` and it's somehow a naming convention in golang to name the self reference only by the first letter of the type